frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -60,6 +61,20 @@ export class SmartScalar extends Component {
   static noHeader = true;
 
   static settings = {
+    ...fieldSetting("scalar.field", {
+      title: t`Field to show`,
```

Comment:
I think we need to add an `isValid` check to handle the case when `scalar.field` refers the column that does not exist anymore

Reply:
turns out that `fieldSettings` already implements this:
https://github.com/metabase/metabase/blob/6091adcab68be9e4ae29facedfd8ec29af521894/frontend/src/metabase/visualizations/lib/settings/utils.js#L41-L42